ilc/server/app.js

```diff
@@ -160,6 +160,9 @@ module.exports = (registryService, pluginManager, context) => {
 
         const route = req.raw.router.getRoute();
 
+        console.log('route = ', route);
```

Comment:
Should be replaced with logger or removed.

Reply:
already done